canvas_modules/common-canvas/src/common-properties/controls/toggletext/toggletext.scss

```diff
@@ -29,11 +29,19 @@
 			border-bottom: 2px solid $support-03;
 		}
 	}
+	button svg {
+		height: $spacing-04;
+		padding-left: $spacing-02;
+	}
 }
 
 // toggle control in table needs to fit within column
 .properties-table-cell-control {
 	.properties-toggletext button {
 		padding: 0 $spacing-02;
+		svg {
```

Comment:
This is not needed as the css for the `.properties-toggletext` also applies to those inside tables